config/splat.us.stno0.yaml

```diff
@@ -39,6 +39,9 @@ segments:
     subalign: 4
     subsegments:
       - [0x0, data]
+      - [0x778, data, e_laydef]
+      - [0x91C, data, e_init]
+      - [0xC8C, data]
```

Comment:
Wasn't technically required for this PR I don't think but I split these while working out the different addresses and lining up with what's in other stage overlays